src/__tests__/buildFilesData/builtOA3.json

```diff
@@ -162,6 +197,11 @@
         "summary": "weather data",
         "description": "Create a new weather record.",
         "operationId": "weatherIdIdPut",
+        "parameters": [
```

Comment:
Not directly relevant, but I was copy/pasting the spec to ensure it was valid into a linter and it pointed out this omission.